lib/ansible/modules/cloud/amazon/dynamodb_table.py

```diff
@@ -411,9 +462,20 @@ def main():
     except (NoAuthHandlerFound, AnsibleAWSError) as e:
         module.fail_json(msg=str(e))
 
+    if module.params.get('tags'):
+        try:
+            region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
+            boto3_dynamodb = boto3_conn(module, conn_type='client', resource='dynamodb', region=region, endpoint=ec2_url, **aws_connect_kwargs)
+            boto3_sts = boto3_conn(module, conn_type='client', resource='sts', region=region, endpoint=ec2_url, **aws_connect_kwargs)
```

Comment:
If you really think I should change the module to no longer depend on boto, I will do it, but I'd much rather do it is a separate pull request. It is a significant change to the existing module, and I will have to retest all of the functionality.

I added the check to ensure that boto3 has DynamoDB tagging support when tagging is specified and retested